test/data/manifests/rhel_84-ppc64le-qcow2-boot.json

```diff
@@ -11231,7 +11231,6 @@
         "/proc": ".M.......",
         "/run/cockpit": ".M.......",
         "/sys": ".M.......",
-        "/var/lib/powerpc-utils/smt.state": "..5....T.",
```

Comment:
hmm, this change looks a bit weird 🤔

Reply:
Yeah, no idea what's going on there.  Is it better that it's not modified?  Or is this one of the files that's expected to be modified?  Is it possible the install order affects this?  I have no idea.

Reply:
the `smt.state` is something host specific so it is definitely better if it is not in the image; the question is, why was it there before? I guess we leak build host state into the image.

Reply:
i.e.:
```
/usr/sbin/smtstate: script is used to persist the current SMT state and to restore it at a convenient time. The idea behind it is to use the script together with a systemd service to save SMT state at any shutdown event and to restore it at an early stage of the boot process. Such implementation creates a way to persist the SMT state across reboots, without requiring any other configuration from system administrators.
```

Reply:
Seems that the file failing `rpm --verify` was a bug
https://bugzilla.redhat.com/show_bug.cgi?id=2014916
Though it's weird that it changed here since the package didn't change in the image.  I think you might be right @gicmo that it could be leaking from the host.

Reply:
I think we should consider removing that file from the image if it was created, since the state of the build host's really should not be in the image and maybe even have an image-info test to report if it is in the image or not.

Reply:
> Seems that the file failing `rpm --verify` was a bug https://bugzilla.redhat.com/show_bug.cgi?id=2014916 Though it's weird that it changed here since the package didn't change in the image. I think you might be right @gicmo that it could be leaking from the host.

The reason for it disappearing is most probably the fact that we are running `rpm` from the host when generating image-info report.